### DISCUSSION TITLE
vacuum.send_command:  inserting {{msg.RoomID}}  (using Node Red)

### DISCUSSION THREAD

**Discussion by c-u-b-e** [Tasshack/dreame-vacuum#354] [Q&A] [answered]
Hi, 

I would like to insert {{msg.RoomID}} in the code:

```
{
   "command":"action",
   "params":{
       "did":"6.2",
       "siid":6,
       "aiid":2,
       "in":[
           {
               "piid":4,
               "value":"{\"customeClean\":[[{{msg.RoomID}},2,4,3,0]]}"
           }
       ]
   }
}
```

But it doesn't work. Do you have any ideas?

Thanks a lot!

![2024-01-02_16h20_14](https://github.com/Tasshack/dreame-vacuum/assets/6537074/077049ed-6c18-47dc-a387-54c7f91275cc)
![2024-01-02_16h43_50](https://github.com/Tasshack/dreame-vacuum/assets/6537074/a5b92762-2670-4d62-807e-0db1f82f76c4)

**Comment by Tasshack** [comment, marked as answer]
You cannot use `customeClean` command for updating just a single room settings and instead you have to pass all rooms in the selected map that's why service call won't work even if you can pass the `RoomID`.
